File: demo-gigya-java/src/main/java/com/ownid/demo/gigya/OwnIdGigyaAccount.java
package com.ownid.demo.gigya;

import com.gigya.android.sdk.account.models.GigyaAccount;
import com.google.gson.JsonObject;

import org.jetbrains.annotations.Nullable;

/**
 * Gigya user account class used in this demo.
 */
public class OwnIdGigyaAccount extends GigyaAccount {

    /**
     * Contains OwnID data
     */
    @Nullable
    private JsonObject data;

    public OwnIdGigyaAccount(@Nullable JsonObject data) {
        this.data = data;
    }

    @Nullable
    public final JsonObject getData() {
        return this.data;
    }

    public final void setData(@Nullable JsonObject data) {
        this.data = data;
    }
}

File: demo-gigya/src/main/java/com/ownid/demo/gigya/OwnIdGigyaAccount.kt
package com.ownid.demo.gigya

import com.gigya.android.sdk.account.models.GigyaAccount
import com.google.gson.JsonObject

/**
 * Gigya user account class used in this demo.
 *
 * @param data  contains OwnID data
 */
class OwnIdGigyaAccount(var data: JsonObject? = null) : GigyaAccount()